bin/firecrown

```diff
@@ -14,21 +14,26 @@ def run(action, config):
     Also, run some ACTION on a CONFIG file."""
     config, data = firecrown.parse(config)
     print("Watch out! Here comes a firecrown!")
-    print("config file:\n", pprint.pformat(config))
 
     # FIXME: stubby code for testing
     if action == 'compute':
         params = {}
         for p in ['Omega_k', 'Omega_b', 'Omega_c', 'h',
-                  'n_s', 'sigma8', 'w0', 'wa']:
+                  'n_s', 'A_s', 'w0', 'wa']:
             if p in config['parameters']:
-                params[p] = config['parameters'][p]
+                val = config['parameters'][p]
+                if isinstance(val, list):
+                    val=val[1]
```

Comment:
should we be more careful here? Also, pls put spaces around operators. I am surprised flake8 didn't complain.

Reply:
now checked for correct length.
also added spaces.